src/Middleware/OutputCaching/src/OutputCacheAttribute.cs

```diff
@@ -0,0 +1,91 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Microsoft.AspNetCore.OutputCaching;
+
+/// <summary>
+/// Specifies the parameters necessary for setting appropriate headers in output caching.
+/// </summary>
+[AttributeUsage(AttributeTargets.Class | AttributeTargets.Method, AllowMultiple = false, Inherited = true)]
+public class OutputCacheAttribute : Attribute
```

Comment:
Should this be sealed?

```suggestion
public sealed class OutputCacheAttribute : Attribute
```